go.mod

```diff
@@ -27,73 +27,70 @@ require (
 	github.com/elazarl/goproxy/ext v0.0.0-20190911111923-ecfe977594f1 // indirect
 	github.com/emicklei/go-restful v2.10.0+incompatible // indirect
 	github.com/ghodss/yaml v1.0.0
-	github.com/go-bindata/go-bindata v3.1.1+incompatible
+	github.com/go-bindata/go-bindata v3.1.2+incompatible
 	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
-	github.com/golang/groupcache v0.0.0-20191002201903-404acd9df4cc // indirect
-	github.com/golang/mock v1.3.1 // indirect
 	github.com/golangci/golangci-lint v1.18.0
-	github.com/google/go-cmp v0.3.1
+	github.com/google/go-cmp v0.4.0
 	github.com/google/renameio v0.1.0
-	github.com/googleapis/gnostic v0.3.1 // indirect
 	github.com/gostaticanalysis/analysisutil v0.0.3 // indirect
 	github.com/hashicorp/golang-lru v0.5.3 // indirect
 	github.com/huandu/xstrings v1.2.0 // indirect
 	github.com/imdario/mergo v0.3.9
 	github.com/magiconair/properties v1.8.1 // indirect
 	github.com/opencontainers/go-digest v1.0.0
-	github.com/openshift/api v3.9.1-0.20191111211345-a27ff30ebf09+incompatible
-	github.com/openshift/client-go v0.0.0-20200320150128-a906f3d8e723
-	github.com/openshift/library-go v0.0.0-20200320155611-2a351bebf158
-	github.com/openshift/runtime-utils v0.0.0-20191011150825-9169de69ebf6
+	github.com/openshift/api v0.0.0-20200829102639-8a3a835f1acf
```

Comment:
Is the version downgrade due to the update not being officially released yet?

Reply:
the versions are misleading, this is newer than the previous one 20191111211345 vs 20200829102639